Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine3.11
```

Comment:
I still don't get why we need a Dockerfile here after all.

Reply:
Well we didn't need it in previous version of docker-compose, now instead of getting image we are building it trought dockerfile

Reply:
Okay, but why?

Reply:
Because I'm not sure how Docker builds container based on image, but with Dockerfile I know exact steps that it will go trough. It might be unnecessary but for now it's easier for me to grasp the concept

Reply:
We don't need own build, so let's use image with volume. In this Dockerfile you do nothing special. We will run these commands (npm install, npm run serve) manually inside container. That's all.

Reply:
Okay then, I switched to docker-compose image build.